src/ngraph/op/max.cpp

```diff
@@ -52,6 +52,8 @@ shared_ptr<Node> op::Max::get_default_value() const
         return make_constant_from_string("0", get_element_type(), get_shape());
     case element::Type_t::bf16:
         return make_constant_from_string("-INFINITY", get_element_type(), get_shape());
+    case element::Type_t::f16:
+        return make_constant_from_string("-INFINITY", get_element_type(), get_shape());
```

Comment:
nit: looks like all these bf/f cases can be simplified to fall into a single `return make_constant_from_string("-INFINITY", get_element_type(), get_shape());` statement?

Reply:
Fixéd.